src/components/Checkbox/index.tsx

```diff
@@ -24,6 +25,9 @@ export const Checkbox = (props: TCheckboxProps): JSX.Element | null => {
     afterLink
   } = props
 
+  const localRef = useRef(null)
+  const inputRef = ref && Object.keys(ref).length ? ref : localRef
```

Comment:
Is this line code about forward ref warning?

Reply:
Not about the warning, about the error